### DISCUSSION TITLE
[✅ DONE] Option to disable top articles on frontpage.

### DISCUSSION THREAD

**Discussion by wjpvandenheuvel** [auroral-ui/hexo-theme-aurora#66] [Ideas]
Especially when starting a new blog showing articles, top articles and featured articles on the front page can be a bit too much. When the site only contains one post (when starting a new blog), the top articles section will be empty and not load. Perhaps it is an option to disable/enable 'top articles' in the theme config or introduce the 'top' variable that can be set to either true or false just like the 'feature' variable.

When no 'top' posts are defined, the top articles section should not load.

**Comment by bennyxguo** [comment]
First of all, thanks for your ideas! 

Yes, for the starter, there aren't much posts to fill out the blog.There should be an option to disable or perhaps a logic to hide the top/feature post section. 

To be more customizable, I think there should be these options:
1. Can disable the feature section totally through `theme` / `layout` config option.
2. If no `feature` properties are set in any articles, the feature article section will be hidden. 
3. Add a layout specially for the condition when there are no feature articles. (what I am thinking is use the first article as the horizontal article layout, then the rest are in grid layout, this way the whole visual harmony is retained.)

**Reply by HarryWangATX** [reply]
would also be great, if there can be a layout without the cover images!

**Reply by bennyxguo** [reply]
> would also be great, if there can be a layout without the cover images!

This idea is also a plus, I got a few request from users, where they don't need a cover page. Will plan on this in later versions.

**Comment by wjpvandenheuvel** [comment]
Great suggestions TriDiamond! I'm very new to Hexo, but have been using your themes ever since.
Kudo's for a great theme and your commitment.

**Reply by bennyxguo** [reply]
@wjpvandenheuvel This feature is shipped in version [1.5.0](https://github.com/auroral-ui/hexo-theme-aurora/releases/tag/v1.5.0)

**Reply by wjpvandenheuvel** [reply]
> @wjpvandenheuvel This feature is shipped in version [1.5.0](https://github.com/auroral-ui/hexo-theme-aurora/releases/tag/v1.5.0)

Works like a charm! Thanks for your hard work.
